src/test/java/com/desiato/puresynth/BaseTest.java

```diff
@@ -0,0 +1,38 @@
+package com.desiato.puresynth;
+
+import com.desiato.puresynth.configurations.AuthTokenFilter;
+import com.desiato.puresynth.repositories.SessionRepository;
+import com.desiato.puresynth.repositories.UserRepository;
+import com.desiato.puresynth.services.AuthenticationService;
+import com.desiato.puresynth.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.web.servlet.MockMvc;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class BaseTest {
```

Comment:
Ideally a separate, small PR.

Reply:
ok, but in a situation like that. should I create a small pr braching off the pr we are working at the moment?

Reply:
You should create a new PR and merge it to the main branch. I would just copy those changes using `cmd + c`.